cfme/configure/configuration/__init__.py

```diff
@@ -515,6 +515,47 @@ def update(self):
         self.appliance.server_details_changed()
 
 
+class VMwareConsoleSupport(Updateable, Pretty, Navigatable):
+    """
+    This class represents the "VMware Console Support" section of the Configuration page.
+    Note this is to support CFME 5.8 and beyond functionality.
+
+    Args:
+        console_type: VMware VMRC Plugin, VNC or VMware WebMKS
```

Comment:
more like `:py:class:String`, which I thought there would actually be examples of. Looks like we only use that with :py:class:<specific class>.  At least state that its a string matching one of those. If you have the class constant for CONSOLE_TYPE you can reference it.